Makefile

```diff
@@ -81,10 +80,5 @@ $(MERGED_CONFIG): clean $(MERGE_CMD) $(CONFIG_FILES)
 		$(shell for o in $(ORGS); do echo "--org-part=$$o=config/$$o/org.yaml"; done) \
 		> $(MERGED_CONFIG)
 
-$(PROW_PATH):
-	mkdir -p $(PROW_PATH)
-	git clone --depth=1 https://github.com/kubernetes-sigs/prow $(PROW_PATH)
-
-$(PERIBOLOS_CMD): $(PROW_PATH)
-	cd $(PROW_PATH) && \
-		go build -o $(PERIBOLOS_CMD) ./prow/cmd/peribolos
+$(PERIBOLOS_CMD):
+	GOBIN=$(OUTPUT_BIN_DIR) go install sigs.k8s.io/prow/cmd/peribolos@latest
```

Comment:
note: this actually means the most recent tag, if you want to continue installing the _latest_ code (equivalent to cloning without specifying a version) you need to do `@$branch`

https://go.dev/ref/mod#version-queries
https://go.dev/ref/mod#pseudo-versions

prow isn't tagged yet, but when/if it starts tagging this will change behavior following the PR here